### PR TITLE
Add User Access tab to credentials, dec env, project and rulebook actv

### PR DESCRIPTION
**Pull request by ZitaNemeckova**
- Create a common component `Access` and pass params based on User/Team component.
- Add User Access tab to credentials, dev. env., project and rulebook activation

### CODE REVIEW

File: frontend/eda/access/common/TeamAccess.tsx [hunk 1]
```diff
@@ -1,176 +1,26 @@
...
+          sort: 'team__name',
+          label: t('Team name'),
+        },
+      }}
+      toolbarFiltersValues={{ label: '', query: '' }}
+      url={edaAPI`/role_team_assignments/`}
+      id={id}
+      content_type_model={type}
+      addRolesRoute={addRolesRoute}
+      translatedType={t('team')}
```
Comment (himdel) [inline frontend/eda/access/common/TeamAccess.tsx:23]: these params look great! :+1:  (well, hub namespaces will need a custom onSave but that's a problem for later :))

but... `translatedType` not so much .. plurals won't work, and neither will any type of declinations

What about passing in `'team' | 'user'` (not translated) and having the full strings and

```diff
-        label: t('Add {{type}}', { type: props.translatedType }),
+        label: translatedType === 'user' ? t`Add user` : t`Add team`,
```

etc

Comment (ZitaNemeckova) [reply frontend/eda/access/common/TeamAccess.tsx]: Fixed

File: frontend/eda/access/common/TeamAccess.tsx [hunk 2]
```diff
@@ -1,176 +1,26 @@
...
-      defaultSubtitle={t('Team access')}
+    <Access
+      tableColumnFunctions={{
+        name: {
+          function: (teamAccess: TeamAssignment) => teamAccess.summary_fields.team.name,
+          sort: 'team__name',
+          label: t('Team name'),
+        },
+      }}
+      toolbarFiltersValues={{ label: '', query: '' }}
```
Comment (vidyanambiar) [inline frontend/eda/access/common/TeamAccess.tsx:18]: The `Team Access` filters do not appear to be working. We probably need to provide initial values here? 
`toolbarFiltersValues={{ label: t('Team name'), query: 'team__name' }}`

**Review by vidyanambiar — COMMENTED** [review]
I added a comment about filters. Other than that this looks great to me 👍

File: frontend/eda/access/common/TeamAccess.tsx (hunk 2 above)
Comment (ZitaNemeckova) [reply frontend/eda/access/common/TeamAccess.tsx]: Fixed. Thanks!

**Review by vidyanambiar — APPROVED** [review]

**Pull request merged**